### DISCUSSION TITLE
Doh hiding domains

### DISCUSSION THREAD

**Discussion by notHiks** [AdguardTeam/AdGuardHome#5683] [Q&A]
Is it expected that DOH requests all show as example.com

![image](https://user-images.githubusercontent.com/66910081/229761182-57b37b9f-c8a5-49fa-a667-10293954759a.png)

if so then how do you filiter these requests ?

**Comment by ainar-g** [comment]
These are most likely test queries from your browser.  At least my Firefox seems to produce lots of these.

**Comment by notHiks** [comment]
i thought so too at first but its 100% of my traffic for DOH

![image](https://user-images.githubusercontent.com/66910081/229797438-225f9ae1-aa38-4c9a-b808-9438c28e90f2.png)

**Comment by irrweg** [comment]
Could be some devices or apps checking if DNS Query are going through.
What does the Log show from which Client those Query come from?

Also some University like the TMU ( http://131.159.24.242/ ) do DNS Tests on Public DNS Servers.
